website/src/components/Edit/InputForm.tsx

```diff
@@ -210,7 +210,16 @@ export class EditableSequences {
         const fastaContent =
             sequences.length === 1
                 ? `>${submissionId}\n${sequences[0].value}`
-                : sequences.map((sequence) => `>${submissionId}_${sequence.key}\n${sequence.value}`).join('\n');
+                : sequences
+                      .map((sequence) => {
+                          if (sequence.value.trim().length > 0) {
+                              return `>${submissionId}_${sequence.key}\n${sequence.value}`;
+                          } else {
+                              return null;
+                          }
+                      })
+                      .filter(Boolean)
```

Comment:
hmm... doesn't filter(Boolean) remove `null` from the array?

Reply:
ah ok this does work on a preview - so perhaps the backend sets them to `null`?

Reply:
ah I see - it should be removed because this produces the fasta - please ignore this